Sources/VLCMediaListPlayer.m

```diff
@@ -205,4 +273,28 @@ - (VLCRepeatMode)repeatMode
 {
     return _repeatMode;
 }
+
+#pragma mark - Delegate methods
+
+- (void)mediaListPlayerPlayed
+{
+    if ([_delegate respondsToSelector:@selector(mediaListPlayerPlayed)]) {
+        [_delegate mediaListPlayerPlayed:self];
+    }
+}
+
+- (void)mediaListPlayerNextItemSet:(VLCMedia *)media
```

Comment:
if this is a set why is there only one media ?

Reply:
The name is a bit confusing but this is mirroring the libvlc API.
We are exposing as `mediaListPlayer:nextMedia:` publicly.